galleries/examples/specialty_plots/leftventricle_bullseye.py

```diff
@@ -104,19 +104,19 @@ def bullseye_plot(ax, data, seg_bold=None, cmap="viridis", norm=None):
 
 # Set the colormap and norm to correspond to the data for which
 # the colorbar will be used.
-cmap = mpl.cm.viridis
+cmap = mpl.colormaps["viridis"]
 norm = mpl.colors.Normalize(vmin=1, vmax=17)
 # Create an empty ScalarMappable to set the colorbar's colormap and norm.
 # The following gives a basic continuous colorbar with ticks and labels.
-fig.colorbar(mpl.cm.ScalarMappable(cmap=cmap, norm=norm),
+fig.colorbar(mpl.cm.ScalarMappable(cmap="viridis", norm=norm),
              cax=axs[0].inset_axes([0, -.15, 1, .1]),
              orientation='horizontal', label='Some units')
 
 
 # And again for the second colorbar.
-cmap2 = mpl.cm.cool
+cmap2 = mpl.colormaps["cool"]
```

Comment:
Also unused and should be deleted.

Reply:
as [above](https://github.com/matplotlib/matplotlib/pull/29028#discussion_r1899111152). Let's leave it like this for now.